inputs/tcp/tcp.go

```diff
@@ -48,19 +48,25 @@ func (l *TcpInput) Run(output chan common.MapStr) error {
   logp.Info("[TcpInput] Running TCP Input")
   server, err := net.Listen("tcp", ":" + strconv.Itoa(l.Port))
   if server == nil {
-      panic("couldn't start listening: " + err.Error())
+      // question - should we really quit completely if we can't bind to a single socket?
+      // or just log an error and keep going
+      panic ("couldn't start listening: " + err.Error())
```

Comment:
I can understand "quit completely" in light of the original "single input only" design, but multiple inputs suggests changing this..  I'll go ahead and change it..